### PR TITLE
Initial integration testing setup

### PR DESCRIPTION
**Pull request by alexzherdev**

### CODE REVIEW

File: desktop/package.json [hunk 1]
```diff
@@ -10,18 +10,24 @@
...
     "gulp-shell": "^0.3.0",
     "gulp-util": "^3.0.4",
+    "mocha": "^3.0.2",
     "node-inspector": "^0.12.8",
     "node-pre-gyp": "^0.6.29",
     "plist": "^1.2.0",
-    "run-sequence": "^1.2.1"
+    "run-sequence": "^1.2.1",
+    "spectron": "^3.3.0",
+    "webpack": "^1.12.14"
```
Comment (gdowens) [inline desktop/package.json]: Don't think we need this dependency anymore?

**Comment by gdowens** [comment]
The only thing that's a bit odd is the use of "class" attributes for selectors. Can we use "id" instead? Just seems clearer and less likely to go horribly wrong.

File: desktop/package.json (hunk 1 above)
Comment (alexzherdev) [inline desktop/package.json]: Yes, most likely a rebase fail

**Comment by alexzherdev** [comment]
It makes sense for the ones in this PR, but if there were multiple instances of the same component on the page, having the same ids would be invalid HTML.

**Comment by gdowens** [comment]
+1

you win the internet

**Pull request merged**